database/frontend/ui-libraries.json

```diff
@@ -0,0 +1,23 @@
+[
+    {
+        "name": "MaterialUI",
+        "url": "https://material-ui.com/",
+        "description": "React components for faster and easier web development. Build your own design system, or start with Material Design.",
+        "category": "frontend",
+        "subcategory": "ui-libraries"
+    },
+    {
+        "name": "ChakraUI",
+        "url": "https://chakra-ui.com/",
+        "description": "Simple, Modular and Accessible UI Components for your React Applications. Built with Styled System",
```

Comment:
```suggestion
        "description": "Simple, Modular, and Accessible UI Components for your React Applications. Built with Styled System",
```

Reply:
can you  please tell me , what changes I need to do.